packages/components/popover/src/popover.js

```diff
@@ -0,0 +1,185 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import '@tradeshift/elements.icon';
+import css from './popover.css';
+
+/**
+ * Position on the X axis.
+ */
+export const PositionX = {
+	LEFT: 'left',
+	RIGHT: 'right'
+};
+
+/**
+ * Position on the Y axis.
+ */
+export const PositionY = {
+	TOP: 'top',
+	BOTTOM: 'bottom'
+};
+
+export class TSPopover extends TSElement {
+	static get styles() {
+		return [TSElement.styles, unsafeCSS(css)];
+	}
+
+	static get properties() {
+		return {
+			/** Is the popover visible or not */
+			opened: { type: Boolean, reflect: true },
+			/** Position on X axis relative to the anchor. Could be 'left', 'right' */
+			positionX: { type: String, attribute: 'position-x', reflect: true },
+			/** Position on Y axis relative to the anchor. Could be 'top', 'bottom' */
+			positionY: { type: String, attribute: 'position-y', reflect: true },
```

Comment:
It's my personal preference but I think having a single attr for positioning/placement is cleaner and easier to handle both in our code and usage of the component.  `topRight`, `topLeft`, `bottomRight`, `bottomLeft` and we can add more options to it later.
I suggest change the name with `placement` as well to not cause confusion with position-left/top attrs.

Reply:
Great suggestion! I also did not like the possible confusion between `positionX` and `positionLeft`. I think it looks better now.